native/src/test/scala/sqs4s/native/ClientSpec.scala

```diff
@@ -0,0 +1,99 @@
+package sqs4s.native
+
+import java.time.Instant
+import java.util.concurrent.TimeUnit
+
+import cats.effect.{Clock, IO}
+import fs2.Stream
+import io.circe.generic.semiauto._
+import io.circe.{parser, _}
+import io.circe.syntax._
+import org.http4s.Uri
+import org.http4s.client.blaze.BlazeClientBuilder
+import org.scalacheck.{Arbitrary, Gen}
+import sqs4s.api._
+import sqs4s.internal.aws4.IOSpec
+import sqs4s.native.serialization.{SqsDeserializer, SqsSerializer}
+
+class ClientSpec extends IOSpec {
+  override implicit lazy val testClock: Clock[IO] = new Clock[IO] {
+    def realTime(unit: TimeUnit): IO[Long] = IO.delay {
+      Instant.now().toEpochMilli
+    }
+
+    def monotonic(unit: TimeUnit): IO[Long] = IO(0L)
+  }
+
+  val accessKey = sys.env("ACCESS_KEY")
+  val secretKey = sys.env("SECRET_KEY")
+  val awsAccountId = sys.env("AWS_ACCOUNT_ID")
+  val queue = Uri.unsafeFromString(
+    s"https://sqs.eu-west-1.amazonaws.com/$awsAccountId/test"
+  )
+  val settings = SqsSettings(queue, AwsAuth(accessKey, secretKey, "eu-west-1"))
+
+  behavior.of("SQS Consumer and Producer")
+
+  case class TestMessage(string: String, int: Int, boolean: Boolean)
+
+  object TestMessage {
+    implicit val encode: Encoder[TestMessage] = deriveEncoder
+    implicit val decode: Decoder[TestMessage] = deriveDecoder
+
+    implicit val desrlz = new SqsDeserializer[IO, TestMessage] {
+      override def deserialize(u: String): IO[TestMessage] =
+        IO.fromEither(parser.decode[TestMessage](u))
+    }
+
+    implicit val srlz = new SqsSerializer[TestMessage] {
+      override def serialize(t: TestMessage): String =
+        t.asJson.noSpaces
+    }
+
+    implicit val arb: Arbitrary[TestMessage] = {
+      val gen = for {
+        str <- Gen.alphaNumStr
+        int <- Gen.choose(Int.MinValue, Int.MaxValue)
+        bool <- Gen.oneOf(Seq(true, false))
+      } yield TestMessage(str, int, bool)
+      Arbitrary(gen)
+    }
+
+    def arbStream(n: Int): Stream[IO, TestMessage] = {
+      val msg = arb.arbitrary.sample.get
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of get on optional type.](https://app.codacy.com/manual/d2a4u/sqs4s/pullRequest?prid=4059007)